packages/data-types/src/types/v1/any.ts

```diff
@@ -0,0 +1,17 @@
+import BaseType from '../base-type';
+
+export default class AnyType extends BaseType {
+    toESMapping(_version?: number) {
+        return {
+            mapping: {}
```

Comment:
I am thinking for this, we should probably use an [unindexed field](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/enabled.html) so that it doesn't get auto indexed by elasticsearch causing weird issues.

```suggestion
            mapping: { [this.field]: { enabled: false }}
```